pyzo/core/kernelbroker.py

```diff
@@ -73,6 +73,9 @@ def __init__(self, info=None):
         # Instantiate with a value that is settable
         self.gui = "Auto"
 
+        # Use IPython if available
+        self.ipython = "yes"
```

Comment:
Why add this?

Reply:
It is not required by the rest of the PR. I just think it is a better default. Currently, when Pyzo autodetects a config, it does not set it to use IPython, so the user has to activate it manually. But if IPython is available on the system, is it not sound to assume it is to use it ?

Reply:
I prefer it not to be the default, because of backwards compat, but also because the integration has some rough edges (running a script, debugging).

Reply:
Ah, that's a little annoying because presently autodetection just will not activate IPython even in a context where this works like a charm.

Maybe the solution would be to not have hardcoded defaults for new environments in __init__ but outsource this to a config template in defaultConfig/siteConfig ? Thus a site admin could tune the template (ie. IPython but for GUI integration there might be some places where "auto" is not the best default).

Reply:
I'm not sure if I follow. Does the new autodetection in this PR behave differently regarding IPython integration? Or do you mean that at your institution you'd prefer to to make check that box by default?